packages/pharos/src/components/popover/pharos-popover.scss

```diff
@@ -0,0 +1,44 @@
+@use '../../utils/scss/mixins';
+
+:host {
+  display: block;
+  position: fixed;
+  z-index: 140;
+  pointer-events: none;
```

Comment:
This is often a red flag for accessibility; I've added @sirrah-tam to review

Reply:
Not sure if desirable, but I stole this from the [dropdown-menu css](https://github.com/ithaka/pharos/blob/develop/packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu.scss#L3-L10)

Reply:
I think as long as the content is inline in the DOM from the triggering element, the use of `z-index` and `position: fixed` seem okay to me. Depending on the visual layout, we just want to make sure it matches the expected reading order.